cmd/promtool/tsdb.go

```diff
@@ -354,15 +354,18 @@ func listBlocks(path string, humanReadable bool) error {
 	if err != nil {
 		return err
 	}
-	printBlocks(blocks, humanReadable)
+	printBlocks(blocks, true, humanReadable)
 	return nil
 }
 
-func printBlocks(blocks []tsdb.BlockReader, humanReadable bool) {
+func printBlocks(blocks []tsdb.BlockReader, writeHeader, humanReadable bool) {
 	tw := tabwriter.NewWriter(os.Stdout, 0, 0, 2, ' ', 0)
 	defer tw.Flush()
 
-	fmt.Fprintln(tw, "BLOCK ULID\tMIN TIME\tMAX TIME\tDURATION\tNUM SAMPLES\tNUM CHUNKS\tNUM SERIES\tSIZE")
+	if writeHeader {
+		fmt.Fprintln(tw, "BLOCK ULID\tMIN TIME\tMAX TIME\tDURATION\tNUM SAMPLES\tNUM CHUNKS\tNUM SERIES\tSIZE")
```

Comment:
Minor issue here: The tabs are only aligned for the first block we print, because subsequent blocks are written with a new tabwriter.

Perhaps we can return the tabwriter and have it as an optional last argument in `printBlocks` (i.e. can be `nil`) so that we can recycle the first tabwriter?

Or something like that.

Reply:
I need to check, but it only prints upon flush.

Reply:
Which defeats the purpose of a live tail of created blocks

Reply:
Flushing multiple times works but does not preserve alignment.

Reply:
So let's accept that they will not be aligned, or maybe do not print the full lie but only the ULID?

Reply:
I think tabs not being aligned is okay but we should print the full line as the developer might want to check if the blocks are created correctly for which, mint, maxt, duration time would be helpful to see.